ui/src/app/modules/domain/column/item-sorter.ts

```diff
@@ -15,7 +15,10 @@
  *  limitations under the License.
  */
 
+import { Thought } from '../thought';
+import { ActionItem } from '../action-item';
+
 export interface ItemSorter {
```

Comment:
It would be cleaner to do something like this:
```
type Item = Thought | ActionItem

export interface ItemSorter {
  active: Item[];
  completed: Item[];
}
```